cl/search/api_serializers.py

```diff
@@ -432,11 +432,41 @@ class Meta:
         )
 
 
-class BaseRECAPDocumentESResultSerializer(DocumentSerializer):
+class NestedMetaSerializer(serializers.Serializer):
```

Comment:
I believe the word 'nested' in the name of this class might be redundant. We have other serializer classes to handle complex objects with varying levels of nesting, but their names don't explicitly mention it. Maybe we can rename it to  "MetaDataSerializer",  What do you think?

Reply:
Yeah, I agree, thanks.
I've applied the suggestion.